src/amo/sagas/users.js

```diff
@@ -13,6 +13,7 @@ import {
   currentUserAccount as currentUserAccountApi,
   deleteUserPicture as deleteUserPictureApi,
   editUserAccount as editUserAccountApi,
+  updateUserNotifications as updateUserNotificationsApi,
```

Comment:
Nit: This just jumped out at me now, I'm not sure why, maybe because I have also been working on an api file. In some other saga files, e.g., collections, we just do `import * as api from 'amo/api/collections';` and then call them in the code like `api.getCollectionDetail`. This seems a bit cleaner than having to create all these aliases.

Reply:
That makes sense.